Firmware/RTK_Surveyor/NVM.ino

```diff
@@ -508,16 +507,25 @@ bool loadSystemSettingsFromFileLFS(char* fileName, Settings *settings)
 {
   File settingsFile = LittleFS.open(fileName, FILE_READ);
   if (!settingsFile)
+  {
+    systemPrintf("settingsFile not found in LittleFS\r\n");
     return (false);
+  }
 
   char line[100];
   int lineNumber = 0;
 
   while (settingsFile.available())
   {
     //Get the next line from the file
-    int n = getLine(&settingsFile, line, sizeof(line)); //Use with SD library
-    //int n = settingsFile.fgets(line, sizeof(line)); //Use with SdFat library
+    int n;
+    if (USE_SPI_MICROSD)
+      n = getLine(&settingsFile, line, sizeof(line)); //Use with SD library
+      //int n = settingsFile.fgets(line, sizeof(line)); //Use with SdFat library
```

Comment:
These two lines look left over from when we were testing between SD and sdFat. Let's remove 524 and the comment on 523 since it seems to be working with sdFat.

Reply:
Agreed!